lib/model/rwfolder.go

```diff
@@ -87,6 +89,8 @@ type rwFolder struct {
 	copiers      int
 	pullers      int
 	shortID      uint64
+	newVersion   protocol.Vector
```

Comment:
These won't do what you expect in the long term, as we process a number of files in parallell. You probably want to extend the `sharedPullerState` instead; although I think it already has both the current and new `FileInfo`s so probably it's already there